backend/codesirius/internal_api/views/hidden_test_bundle.py

```
@@ -0,0 +1,51 @@
+class HiddenTestBundleRetrieveUpdateAPIView(APIView):
+    def get_object(self, bundle_pk: int, problem_pk: int) -> HiddenTestBundle:
+        try:
+            return HiddenTestBundle.objects.get(id=bundle_pk, problem_id=problem_pk)
+        except HiddenTestBundle.DoesNotExist:
+            raise NotFound("Hidden test bundle not found")
+        except Exception as e:
+            logger.exception(e)
+            raise NotFound("Failed to retrieve hidden test bundle")
+
+    def get(
+        self, request: Request, problem_pk: int, bundle_pk: int
```

Ruff (0.8.2)

Unused method argument: `request`

(ARG002)